code/src/pages/products/ProductsPage.tsx

```diff
@@ -29,7 +51,7 @@ const ProductsPage = () => {
         />
         <AppBox className="spa-products-page__info">
           <AppTypography className="spa-products-page__count" component="span">
-            {products?.length || 0}
+            {data?.itemsCount}
```

Comment:
What will happen if data is undefined?

Reply:
Data is undefined during loading, loading fallback logic is above (for now just `loading...` text, but it will be changed soon). I think we will not get `undefined` data and success response at the same time. Even if there will be cases like this for some reason, I think we need to handle it on a global level, inside of `ErrorBoundary` or `Router` error handling logic.